tests/gh-actions/kind-cluster-1-22.yaml

```diff
@@ -1,3 +1,4 @@
+# This configuration file should work with KinD v0.14.0
```

Comment:
Since you changed the name from `kind-1-22.yaml` to `kind-cluster-1-22.yaml` we'll need to update the actions that use this config.

Reply:
Fixed it.